app/views/layouts/application.html.erb

```diff
@@ -11,14 +11,18 @@
   </head>
 
   <body>
+    <% if current_user.user_type == 'administrator' %>
```

Comment:
would reccomend using B5 grid for easy management of display

Reply:
B5 Grid?

Reply:
bootstrap grid